tests/api.c

```diff
@@ -20766,7 +20766,76 @@ static int test_wc_ecc_is_valid_idx (void)
 
 
 } /* END test_wc_ecc_is_valid_idx */
+/*
+ * Testing wc_ecc_get_curve_id_from_oid()
+ */
+static int test_wc_ecc_get_curve_id_from_oid (void)
+{
+    int ret = 0;
+#if defined(HAVE_ECC) && !defined(NO_ECC256)  && !defined(HAVE_SELFTEST) && \
+    !defined(HAVE_FIPS)
+    word32 len = 8;
```

Comment:
Should "len" be representing the length of "oid" here?  If so, can we move this below the "const byte oid[]" line below and set "len = sizeof(oid)" instead?  That will be more clear what "8" represents.